pkg/justification/processor_test.go

```diff
@@ -342,7 +342,7 @@ func TestCreateToken(t *testing.T) {
 					err: fmt.Errorf("Cannot connect to validator"),
 				},
 			},
-			wantErr: "unexpected error from validator \"jira\": Cannot connect to validator",
+			wantErr: "unable to validate request",
```

Comment:
Since we now use error code, we should check error codes in tests as well?

Reply:
Done, updated all relevant test cases